apps/api/pages/api/event-types/by-slug/[slug]/_get.ts

```diff
@@ -0,0 +1,84 @@
+import type { NextApiRequest } from "next";
+
+import { defaultResponder } from "@calcom/lib/server";
+
+import { schemaEventTypeReadPublic } from "~/lib/validations/event-type";
+import { schemaQuerySlug } from "~/lib/validations/shared/querySlug";
+
+import { checkPermissions } from "../../[id]/_get";
+import getCalLink from "../../_utils/getCalLink";
+
+/**
+ * @swagger
+ * /event-types/by-slug/{slug}:
```

Comment:
Renamed the route from "/event-types/bySlug" to "/event-types/by-slug" to follow same case in commit [refactor: event-types/bySlug to event-types/by-slug route](https://github.com/calcom/cal.com/pull/13374/commits/372e1bde94fad306f1ba66cb5a3f730ddb7c1bd0)